### DISCUSSION TITLE
I don't have student email id, not student ID, but I paid a $4000 for a 18 Months bootcamp

### DISCUSSION THREAD

**Discussion by ggorantala** [community/community#61124] [GitHub Education]
### Hi there,

I am a student in Scaler and don't have a student email ID, not a Student ID card. I have payment proofs, student emails every day and course track and dashboard access. How can I claim the student pack?

_No response_

**Comment by ggorantala** [comment]
<img width="1138" alt="Screenshot 2023-07-18 at 12 21 53 PM" src="https://github.com/community/community/assets/18720267/0c812d8a-bb90-4d78-9c26-1be73acc3cc9">

**Comment by ggorantala** [comment]
Can anyone help me with this?

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
